src/button/styles.scss

```diff
@@ -65,7 +65,6 @@
     border-color: map.get($variant, 'disabled-border-color');
     color: map.get($variant, 'disabled-color');
     text-decoration: none;
-    pointer-events: none;
```

Comment:
Removed this property, because it prevents triggering tooltip disabled reason. Click event is prevented on the component level: https://github.com/cloudscape-design/components/blob/main/src/button/internal.tsx#L102